### PR TITLE
Sum json follow (json-symbolic-keys)

### PR DESCRIPTION
**Pull request by AlexKnauth**
Goes with https://github.com/fare/gerbil-poo/pull/3

### CODE REVIEW

File: t/types-test.ss [hunk 1]
```diff
@@ -68,6 +69,10 @@
       (check-equal? (element? Zoth (Zoth-h #(5 8 13))) #t)
       (check-equal? (element? Zoth (Zoth-h #(5 8 13 14))) #f)
       (check-equal? (element? Zoth (Zoth-h #(5 -1 13))) #f)
+      (parameterize ((json-symbolic-keys #t))
```
Comment (fare) [inline t/types-test.ss:72]: You should check the effect of the parameter, not just its non-effect.

Comment (fare) [reply t/types-test.ss:72]: I suggest a macro that checks both sides.

**Review by fare — CHANGES_REQUESTED** [review]

**Comment by AlexKnauth** [comment]
wait I wasn't done

**Comment by AlexKnauth** [comment]
@fare I wasn't done with this, so I made #5 to update it

**Pull request merged**